builtin/credential/chef-node/path_config.go

```diff
@@ -0,0 +1,75 @@
+package chefNode
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfig(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "config",
```

Comment:
I suppose /config/node would also make sense for the per node policy bindings.  And I see /config/client has precedent in the AWS backend so I will go ahead and move under /config/client.